packages/protocol/foundry.toml

```diff
@@ -67,6 +64,7 @@ test = "test/layer2"
 script = "script/layer2"
 out = "out/layer2"
 evm_version = "shanghai"
+fuzz.runs = 200
```

Comment:
should we also add this config to other prifiles?

Reply:
Updated to 
```
[profile.default]
fuzz_runs = 200
```